### PR TITLE
create the com.ibm.ws.jaxws.2.3.webcontainer project

### PR DESCRIPTION
**Pull request by WhiteCat22**
Create the com.ibm.ws.jaxws.2.3.webcontainer project for the jaxws-2.3 feature

#7651

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
green build

File: dev/com.ibm.ws.jaxws.2.3.webcontainer/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+     provide:=javax.servlet.ServletContainerInitializer; \
+     immediate:=true;\
+     properties:="service.vendor=IBM"\
+  
+
+Export-Package: com.ibm.ws.jaxws23.webcontainer
+
+Import-Package: \
+   javax.xml.ws.*;version="[2.2,3)", \
+   org.apache.cxf.*;version="[2.6.2,2.6.3)", \
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.webcontainer/bnd.bnd]: This is neither here nor there but I'm not sure if you changed anything, I just  find it strange there is an import for org.apache.cxf.* but no CXF jars listed in the buildpath

**Comment by skasund** [comment]
L2 message review complete

**Pull request merged**